src/test/java/org/fcrepo/importexport/importer/ImporterTest.java

```diff
@@ -144,8 +165,19 @@ public void testImportBinary() throws Exception {
     }
 
     @Test
+    public void testImportExternalResource() throws Exception {
+        final Importer importer = new Importer(externalResourceArgs, clientBuilder);
+        importer.run();
+        verify(client).put(externalResourceURI);
+        verify(externalResourceBuilder).body(eq(new File(externalFilesDir, "rest/ext1.external")),
+                                eq("message/external-body"));
+        verify(client).put(externalResourceDescriptionURI);
+    }
+
+    @Test
     public void testImportWithoutBinaries() throws Exception {
-        final Importer importer = new Importer(noBinaryArgs, clientBuilder);
+        binaryArgs.setIncludeBinaries(false);
```

Comment:
After some brief discussion on IRC, I think that the order the tests run in is dependent on the JVM that's running them.  So please make another copy of the config instead of updating it in the test.